tools/scylla-nodetool.cc

```diff
@@ -956,6 +964,26 @@ Fore more information, see: https://opensource.docs.scylladb.com/stable/operatin
             },
             listsnapshots_operation
         },
+        {
+            {
+                "rebuild",
+                "Rebuilds a node’s data by streaming data from other nodes in the cluster (similarly to bootstrap)",
+R"(
+Rebuild operates on multiple nodes in a Scylla cluster. It streams data from a
+single source replica when rebuilding a token range. When executing the command,
+Scylla first figures out which ranges the local node (the one we want to rebuild)
+is responsible for. Then which node in the cluster contains the same ranges.
+Finally, Scylla streams the data to the local node.
```

Comment:
This is likely inaccurate with RBNO (@asias) but can be corrected later if you just copied it from the docs.

Reply:
Yes, I copied from the docs.